src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const sourceObject = {};
```

Comment:
[CODE STYLE] - The variable name `sourceObject` can be more descriptive. It's a good practice to give meaningful names to your variables. For example, `cssStylesObject` would be a better name in this context.

Reply:
Ok, thanks, I just thought we have to give names, that its name had some references to the function declared statement